File: PostoGasolina.Business/Models/Cliente.cs
using System;
using System.Collections.Generic;

namespace PostoGasolina.Business.Models
{
    public class Cliente : Entity
    {
        public string Nome { get; set; }
        public string Documento { get; set; }
        public DateTime DataCadastro { get; set; }
        public IEnumerable<Veiculo> Veiculos { get; set; }
        public IEnumerable<Abastecimento> Abastecimentos { get; set; }
        public bool Ativo { get; set; }

    }
}

File: PostoGasolina.App/ExtJS/app/model/Cliente.js
Ext.define('PostoGasolina.model.Cliente', {
    extend: 'Ext.data.Model',

    fields: [
        {name:'id', type: 'guid'},
        {name:'nome', type: 'string'},
        {name: 'documento', type: 'string'},
        {name:'dataCadastro', type: 'date'},
        {name:'ativo', type: 'bool'}
    ]

});